src/main/java/com/bobocode/blyznytsia/bibernate/exception/TransactionException.java

```diff
@@ -0,0 +1,26 @@
+package com.bobocode.blyznytsia.bibernate.exception;
+
+/**
+ * Indicates that a transaction could not be begun, committed or rolled back.
+ */
+public class TransactionException extends BibernateException {
```

Comment:
I think it will be better if we'll have error message patterns inside exception.
Like we did here: https://github.com/bobocode-blyznytsia/bring-framework/blob/main/src/main/java/com/bringframework/exception/NoSuchBeanException.java

Reply:
Ok, we have three different messages like
"Error occurred while transaction beginning"
"Error occurred while transaction rollback"
"Error occurred while transaction committing"

Do you suggest creating message patterns like 
`String MSG = "Error occurred while transaction %s";`
and pass the rest of the message while throwing the exception?
for me, it looks strange
`throw new TransactionException ("beginning", e);`